pkg/asset/manifests/proxy.go

```diff
@@ -158,6 +158,13 @@ func createNoProxy(installConfig *installconfig.InstallConfig, network *Networki
 
 	// TODO: IBM[#95]: proxy
 
+	if platform == azure.Name {
+		if installConfig.Azure.CloudName == azure.StackCloud {
+			set.Insert("168.63.129.16")
+		}
+		set.Insert(installConfig.Config.Azure.ARMEndpoint)
```

Comment:
Isn't this only for Azure Stack? Right now it's adding this for Azure too

Reply:
The ArmEndpoint can be for either

Reply:
Code comments say otherwise. Mind changing them then?
https://github.com/openshift/installer/blob/c623779b6c2bbe300723475ad509757283844a5b/pkg/types/azure/platform.go#L31

Reply:
Here's another one.
https://github.com/openshift/installer/blob/c623779b6c2bbe300723475ad509757283844a5b/pkg/asset/installconfig/azure/metadata.go#L22